packages/bsky/src/views/index.ts

```diff
@@ -145,6 +158,13 @@ export class Views {
     return false
   }
 
+  noUnauthenticatedPost(viewer: HydrateCtx['viewer'], post: PostView): boolean {
+    const isNoUnauthenticated = !!post.author.labels?.find(
```

Comment:
I tend to use `some()` in these cases when just checking for existence and the item itself isn't needed.